src/main/webapp/app/exam/shared/exam-shared.module.ts

```diff
@@ -3,10 +3,12 @@ import { StudentExamWorkingTimeComponent } from 'app/exam/shared/student-exam-wo
 import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
 import { TestexamWorkingTimeComponent } from 'app/exam/shared/testExam-workingTime/testexam-working-time.component';
 import { WorkingTimeControlComponent } from 'app/exam/shared/working-time-update/working-time-control.component';
+import { ExamLiveEventComponent } from 'app/exam/shared/events/exam-live-event.component';
+import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
 
 @NgModule({
-    imports: [ArtemisSharedCommonModule],
-    declarations: [StudentExamWorkingTimeComponent, TestexamWorkingTimeComponent, WorkingTimeControlComponent],
-    exports: [StudentExamWorkingTimeComponent, TestexamWorkingTimeComponent, WorkingTimeControlComponent],
+    imports: [ArtemisSharedCommonModule, ArtemisMarkdownModule],
```

Comment:
This causes the build to fail due to an exceeded bundle size. Is there no dedicated module just for the exam management? This should not be needed in the shared module.

Reply:
Hm, unfortunately this markdown module exports the `HtmlForMarkdownPipe` which we need to render the instructor announcements ..

Reply:
[Worked around](https://github.com/ls1intum/Artemis/pull/7254/commits/829f5563414edd9a57132e2b3f7084c968a85e21) this issue by removing `ArtemisExamSharedModule` from `ArtemisCoursesModule`'s imports and moving affected components into their own lazily loaded module.